### ISSUE TITLE
Loss function of language scorer

### ISSUE THREAD

**Issue by kariminf** [kariminf/ML2ExtraSum#2]
Since the batch is a document, the language score between sentences will be the same since language scorer feed is document-dependent no sentence-dependent. 

But, the score must be the same even between different documents of the same language. 
This is why a variable must be used to save last language-score and compare it with the new one.

**Comment by kariminf** [comment]
Gives baaaaaaaaaad results for languages clustering. Back to the simple solution: no document knows of its language-like documents during training. 

It gives some good results :+1: 
![figure_1-2](https://user-images.githubusercontent.com/4449683/43212521-11fb9840-902c-11e8-9626-6afe3c510ee0.png)

**Comment by kariminf** [comment]
I did it wrong. 

Once a model is initialized, it will take the initial values. I used non tensorflow elements: string and hash to control the flow. But, instead, I have to incorporate these in the model

**Comment by kariminf** [comment]
After doing it right (this time); languages scores converge to zero. 

Another criterion must be added: different languages must be divergent 

![figure_1](https://user-images.githubusercontent.com/4449683/43261190-c44eb368-90d3-11e8-956c-2eb96df40a7d.png)

**Comment by kariminf** [comment]
Adding a diverging function; doesn't help 

I think I let the system learn as it pleases it

![figure_1](https://user-images.githubusercontent.com/4449683/43281757-634c50b4-910c-11e8-8989-6f07e1674424.png)
